src/datasets_raw.md

```diff
@@ -0,0 +1,25 @@
+| name                                    |   nb_subjects | has_participant_tsv   | has_participant_json   | participant_columns                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | has_phenotype_dir   | modalities               | sessions                 | tasks                                                                                                                                                                                                                                           |   fmriprep |   freesurfer |   mriqc |
```

Comment:
for the moment I would drop the columns :
- has_participant_tsv
- has_participant_json
- has_phenotype_dir
- participant_columns